go/test/endtoend/vreplication/vreplication_test.go

```diff
@@ -213,7 +239,20 @@ func TestVReplicationDDLHandling(t *testing.T) {
 	waitForWorkflowState(t, vc, ksWorkflow, binlogdatapb.VReplicationWorkflowState_Running.String())
 	// Confirm new col was dropped on target
 	waitForQueryResult(t, vtgateConn, targetKs, checkColQueryTarget, "[[INT64(0)]]")
+	// Confirm that we updated the stats on the target tablet as expected.
```

Comment:
The code that checks for the stats is pretty similar, so might be useful to add a helper function rather than cut/pasting.

Reply:
Good idea. Done here: https://github.com/vitessio/vitess/pull/15769/commits/cbe3a55977cd82a34a17a990bd0380307b0c818b